helm/values-eks.yaml

```diff
@@ -244,10 +240,10 @@ configmap:
         allowedHeaders:
           - "Content-Type"
 
-    task_type_whitelist:
-      spark:
-        - project: flytetester
-        - project: spark-workflows
+  task_type_whitelist:
```

Comment:
I think we can just delete this block. there's no reason to gate spark anymore, ,right?